tests/selenium/pbtest.py

```diff
@@ -259,7 +259,10 @@ def run(self, result=None):
                 if i == nretries - 1:
                     raise
                 else:
-                    print('\nRetrying {} ...'.format(result))
+                    wait_secs = 2 ** i
+                    print('\nRetrying {} after {} seconds ...'.format(
+                        result, wait_secs))
+                    time.sleep(wait_secs)
```

Comment:
Sorry, could you explain what you mean? I intend to have an increasing wait time inbetween retrying failed tests that are marked for retrying:
```
Retrying <TEST_NAME> after 1 seconds ...
Retrying <TEST_NAME> after 2 seconds ...
Retrying <TEST_NAME> after 4 seconds ...
Retrying <TEST_NAME> after 8 seconds ...
Retrying <TEST_NAME> after 16 seconds ...
FAILED
```
Will this code not do that?